indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -82,46 +89,63 @@ def __table_args__(cls):
             lazy=True
         )
     )
-    #: Data stored in abstract/contribution fields
-    field_values = db.relationship(
-        'AbstractFieldValue',
+    track = db.relationship(
+        'Track',
         lazy=True,
-        cascade='all, delete-orphan',
+        foreign_keys=[track_id],
```

Comment:
just fyi, you can omit the `[]` if it's only a single column (or keep them, doesn't really matter)